src/components/PostDetails.tsx

```diff
@@ -1,117 +1,133 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { Post } from '../types/Post';
+import { Comment } from '../types/Comment';
+import { deleteComment, getPostComments } from '../api/PostsApi';
+
+type Props = {
+  post: Post;
+};
+
+export const PostDetails: React.FC<Props> = ({ post }) => {
+  const [error, setError] = React.useState(false);
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [comments, setComments] = React.useState<Comment[]>([]);
+  const [isCommentFormOpen, setIsCommentFormOpen] = React.useState(false);
+
+  const deleteCommentClickHandler = (comment: Comment) => {
```

Comment:
pls read this [article](https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) and try to follow naming convention.  check your handler everywhere